### DISCUSSION TITLE
makro.cfg should be renamed to macro.cfg

### DISCUSSION THREAD

**Discussion by bimawa** [1coderookie/Klipper4Kobra2series#4] [General]
Does it look like a mistake?  Or is this for a special?

**Comment by 1coderookie** [comment]
Which cfg exactly are you referring to?

**Reply by bimawa** [reply]
file makro.cfg I think we shoud rename it to `macro.cfg`, and link in printer.cfg change to `[include macro.cfg]`
in my case  for Kobra2Neo

**Reply by 1coderookie** [reply]
Again, for the next time, please mention the specific file / printer model, because I have several subdirectories for several K2 models here.  
But I also found this one.  
I actually ended up deleting the makro.cfg file (and renaming makro.cfg to macro.cfg, but commented it anyway), because, again, it wasn't mine and I decided to only offer machine-specific printer.cfg files for a basic setup to get the machine up&running.  
Everything else beyond this, like finetuning the cfg, adding specific macros and so on has to be done by every user himself.  
This is not because I don't like to offer more, it's just that I made the experience (with my Klipper4KobraGoNeo repository) that users reach out about minor stuff, asking me to change this and that, compile an up-to-date bin file and so on. 
Besides, users who contributed cfgs or macros in the past didn't bother to maintain the files and e.g. remove deprecated functions etc, so in the end I had to do all that as well.  
And unfortunately I just don't have the time to do so, my infosites are already way too time consuming already.. ;)
